### PR TITLE
Improved time helpers.

### PR DESCRIPTION
**Pull request by ochococo**

### CODE REVIEW

File: Tests/SwiftyTimerTests/SwiftyTime.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+private let hoursInDay: Double      = 24
+
+extension Double {
+    
+    private func assertOneOrLess(value: Double) -> Double {
+        assert(self <= 1, "🤓 Use plural property for numbers above 1.")
+        return value
+    }
+    
+    public var milisecond:  NSTimeInterval { return assertOneOrLess(miliseconds) }
```
Comment (radex) [inline Tests/SwiftyTimerTests/SwiftyTime.swift]: typo, should be "millisecond" — same in other places

File: Tests/SwiftyTimerTests/SwiftyTime.swift [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+private let secondsInMinute: Double = 60
+private let minutesInHour: Double   = 60
+private let hoursInDay: Double      = 24
+
+extension Double {
+    
+    private func assertOneOrLess(value: Double) -> Double {
+        assert(self <= 1, "🤓 Use plural property for numbers above 1.")
+        return value
+    }
```
Comment (radex) [inline Tests/SwiftyTimerTests/SwiftyTime.swift]: I mean, do we really need this? ;)

Comment (ochococo) [inline Tests/SwiftyTimerTests/SwiftyTime.swift]: No but do you really wanna see monsters like `4.day` or `10.second`?

**Comment by radex** [comment]
OK

**Comment by narfdotpl** [comment]
http://i1.kym-cdn.com/photos/images/newsfeed/000/082/456/Okay.png

**Pull request merged**